docs/reference/gqlalchemy/transformations/importing/loaders.md

```diff
@@ -44,7 +43,7 @@ Many to many mapping is intended to be used in case of associative tables.
 - `foreign_key_from` - Describes the source of the relationship.
 - `foreign_key_to` - Describes the destination of the relationship.
 - `label` - Label to be applied to the newly created relationship.
-- `parameters` - Parameters that will be added to the relationship created from this object (Optional).
+- `parameters` - List of parameters that will be added to the relationship created from this object (Optional).
```

Comment:
In the code, you referred to this as Properties instead of a List of parameters. Which one is the correct one? Maybe List of properties?

Reply:
Until now, the `parameters` variable was actually just ignored.
So let's just "break" the api and rename this to `properties`.
Agree?

Reply:
I vote 'yes' 😄